languages/python/exercises/concept/string-methods/.meta/str_methods.py

```diff
@@ -0,0 +1,13 @@
+def capitalize_title(title):
```

Comment:
This file should be named "example.py", and should be in the string-methods/ instead of .meta/ directory.